doc/user/core/trees/ball_tree.md

```diff
@@ -433,7 +431,7 @@ mlpack::data::Load("cloud.csv", dataset, true);
 //
 // Note that the '<>' isn't necessary if C++20 is being used (e.g.
 // `mlpack::BallTree tree(...)` will work fine in C++20 or newer).
-mlpack::BallTree<> tree(std::move(dataset));
+mlpack::BallTree<> tree(std::move(dataset), 10);
```

Comment:
What is the value of 10 we are adding? Did we document it?

Reply:
Yeah, this was just to match the comment above:

```
// Build the ball tree with a leaf size of 10.  (This means that nodes are split
// until they contain 10 or fewer points.)
```

I noticed that I hadn't actually specified a leaf size of 10, so I fixed it.